### ISSUE TITLE
Lenovo N21 not turning on after flashing firmware

### ISSUE THREAD

**Issue by RemiAQ** [MrChromebox/firmware#260]
I installed the custom firmware from the website and installed the firmware correctly; and after telling it to reboot it didnt power back on. The backlight isnt turning on, and the power button isnt doing anything. However my theory is that it just, didnt reboot and turned the screen off and disabled the keyboard. Because theres still a bit of heat coming from it. Any ideas on what to do here? I really dont want to ship it off to mrchrome to get it unbricked, but worse comes to worse im willing to pay the money.

**Comment by MrChromebox** [comment]
try hooking up to an external display, GIMMER is known to have issues with the internal display init

**Comment by RemiAQ** [comment]
> 
> 
> try hooking up to an external display, GIMMER is known to have issues with the internal display init

Nothing is coming up on my ext display. Too late now I guess, already ordered a new chromebook :\

Also; its an engaurde board

**Comment by MrChromebox** [comment]
why order a new Chromebook vs a USB flasher to unbrick it?

**Comment by mattygjak** [comment]
This is where I am at, with a Lenovo N21 Chromebook. I have got other chromebooks to work fine after the running of the firmware script. I ran into a big brick wall with this model Lenovo N21 chromebook, i put in dev mode, opened it up removed the write protect screw and then powered it back on , went in to shell and ran the basic Script from Mrchromebox and it ran, everything looked good backed up the Chrome os, it did its thing and even said Firmware flash was successful, Said that the first time booting may take more time as normal, then I hit enter to reboot and it went black and never turned back on, hiting the power Key did not turn it on, removied the pluged in cord and pluged back in, still nothing, I disconected the battery and left it 5-10 min. and pluged back in, Nothing, no responce at all , wont power on , holding power button and refresh and escape and other combonations , Nothing. it seem to be working so well in the process when flashing the firm ware up till then. looking for help with it on line and landed here, this old post went down the same path .... but with no solution. I am going to order the the USB Flasher to try and unbrick. 
but thought i would post and ask if there is in more information on this model and maybe anything im overlooking or something i could try? I dont know what that could be seen i cant even get it to turn on and try to boot anything. Thank you for any help and for everything eals I have learned from this site and process. this is not my only system and really is just for fun and teach myself stuff so not the end of the world, but would like to sovle this puzzle, is this model able to be flashed firm ware and be able to run a Linus distro with a difrent script or method? is the USB flasher going to be able to return it back to a chromebook and thats it. ?? 

thank you 
you are appreciated.
